src/cockpit/packages.py

```diff
@@ -230,10 +232,14 @@ def add_file(self, item: Path, checksums: List['hashlib._Hash']) -> None:
         # Otherwise, add it to the normal files list (after stripping '.gz').
         po_match = Package.PO_JS_RE.fullmatch(rel)
         if po_match:
-            locale = po_match.group(1)
+            manifest_po = po_match.group(1)
+            locale = po_match.group(2)
             # Accept-Language is case-insensitive and uses '-' to separate variants
             lower_locale = locale.lower().replace('_', '-')
-            self.translations[lower_locale] = data, guessed_type
+            if manifest_po is None:
+                self.translations[lower_locale] = data, guessed_type
+            else:
+                self.manifest_translations[lower_locale] = data, guessed_type
```

Comment:
Feels like it could use a bit more factoring out, but to be honest, I'm not sure I'd like the result...